applications/SwimmingDEMApplication/python_scripts/swimming_DEM_solver.py

```diff
@@ -291,4 +288,10 @@ def ImportModelPart(self): # TODO: implement this
         pass
 
     def GetComputingModelPart(self):
-        return self.dem_solver.spheres_model_part
\ No newline at end of file
+        return self.dem_solver.spheres_model_part
+
+    def GetTimeIntegrationMoveMeshFlag(self):
+        if self.fluid_solver.settings.Has('time_integration_settings') and self.fluid_solver.settings["time_integration_settings"]["move_mesh_flag"].GetBool():
```

Comment:
please, as soon as you construct the the solver class, store the option in a variable self.move_mesh_flag = ..., to make this cleaner and faster (although I know it is not critical here)